### DISCUSSION TITLE
I can't add attributes to a MISP event

### DISCUSSION THREAD

**Discussion by steveS222** [MISP/MISP#9436] [General]
I have created a MISP event and when I tried to add attributes to the event, Getting the following error:
<img width="631" alt="image" src="https://github.com/MISP/MISP/assets/153216655/9cbe4ea4-6f86-4cd7-8fdb-427acd59415e">

Need help here!!

**Comment by Kagee** [comment]
Because reasons, this can happen when your MISP.baseurl is set to something odd/invalid.

**Reply by steveS222** [reply]
Thanks for the reply...!!
So, here is the thing. I installed MISP through docker by following the link https://github.com/misp/misp-docker

I don't see "**MISP.baseurl**" present in either template.env or docker-compose.yml file where we make changes.
The only thing I see here is HOSTNAME like the image below for which I have given my machine IP already.
<img width="295" alt="image" src="https://github.com/MISP/MISP/assets/153216655/e6641a27-2908-45d8-a345-d0e6ca022544">

**Reply by steveS222** [reply]
Hello there!!
Can someone help me out here please
